src/common/configurationReader.ts

```diff
@@ -5,11 +5,45 @@ import {InternalErrorCode} from "./error/internalErrorCode";
 import {ErrorHelper} from "./error/errorHelper";
 
 export class ConfigurationReader {
+    public static readString(value: any): string {
+        if (this.isString(value)) {
+            return value;
+        } else {
+            throw ErrorHelper.getInternalError(InternalErrorCode.ExpectedStringValue, value);
+        }
+    }
+
+    public static readBoolean(value: any): boolean {
+        if (this.isBoolean(value)) {
+            return value;
+        } else if (this.isString(value)) {
+            return value === "true";
```

Comment:
I think that should validate that this is either "true" or "false" and throw an error otherwise